org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -1289,31 +1289,109 @@ class FileEvent {
 /**
  * Value-object describing what options formatting should use.
  */
-@JsonRpcData
-class FormattingOptions {
+class FormattingOptions extends LinkedHashMap<String, Object> {
```

Comment:
We could, but we have already everything in place for `Either` as support in active annotations and JSON parsing.